cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerClusterStatusService.java

```diff
@@ -215,16 +223,37 @@ public ClusterStatusResult getStatus(boolean blueprintPresent) {
     }
 
     @Override
-    public Map<HostName, ClusterManagerState> getExtendedHostStatuses() {
-        Map<HostName, ClusterManagerState> result = new HashMap<>();
-        getHostHealth().forEach((hostname, health) ->
-                result.put(hostname, new ClusterManagerState(healthSummaryToState(health.getSummary()),
-                        getHostHealthMessage(health.getSummary(), health.getExplanation()))));
-        return result;
+    public ExtendedHostStatuses getExtendedHostStatuses() {
+        Map<HostName, Collection<ApiHealthCheck>> hostHealth = getHostHealth();
```

Comment:
Can you please add a few log lines to these methods?

Reply:
added